### PR TITLE
Portfolio header

### PR DESCRIPTION
**Pull request by husush23**
In this branch I did:

-  Edited README file
- Add HTML file
- Add CSS file
- Add linters configuration file
-  Add assets folder

### CODE REVIEW

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @husush23  ,

Great job so far! 
However, you still have minor issues to fix.

May I highlight the following: 
- [x] Your project looks good👏  ✔️ 
- [x] No Linter errors✔️ 


## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <script
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header class="header-mobile">
+      <nav class="nav">
+        <div class="logo"><a href="#"> I'm Hussein</a></div>
+        <div class="menu">
```
Comment (braincee) [inline index.html:24]: - [ ] Good job implementing the required changes by the previous reviewer. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <li>
+              <a href="#"><i class="fa-brands fa-github"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-solid fa-m"></i></a>
+            </li>
+            <li>
+              <a href="#"><img src="./img/Vector.jpg" class="vector" /></a>
+            </li>
+          </ul>
```
Comment (braincee) [inline index.html:59]: - [ ] Good job implementing changes required by the previous reviewer. 👍

File: main.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+/* section in header */
+.introduction {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.greeting-header {
+  /* width: 327px; */
```
Comment (braincee) [inline main.css:60]: - [ ] [OPTIONAL] I kindly suggest removing commented code for clean code.

File: main.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  color: #505f79;
+}
+
+.vector {
+  height: 15px;
+}
+
+/* Media queries */
+
+@media (min-width: 768px) {
```
Comment (braincee) [inline main.css:121]: - [ ] Good job implementing minimum desktop screen size. 👍

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @husush23 ✋, I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅

REQUIRED CHANGES ♻️
------

- [x] Please ensure that there are no linter errors. 😄 
- [ ] Please I suggest adding a mobile version responsiveness (i.e) `375px` per project requirements. 😃 
- [ ] Please I suggest adding responsive values in your CSS rules (i.e) `width: 90%` per project requirements. 😃 

**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

- [ ] [OPTIONAL] I found duplicate README files, please kindly delete one of them. 😃 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @husush23, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**